client/src/components/Message/MessageList.tsx

```diff
@@ -0,0 +1,48 @@
+import { getMessages } from '@/api/messages.api';
+import { Message } from '@/types/Message';
+import notification from '@/utils/notification';
+import { List } from '@mui/material';
+import { FC, useEffect, useState } from 'react';
+import { MessageItem } from './MessageItem';
+
+interface Props {
+  roomId: string;
+}
+
+export const MessageList: FC<Props> = ({ roomId }) => {
+  const [messages, setMessages] = useState<Message[]>([]);
+
+  useEffect(() => {
+    const ws = new WebSocket('ws://localhost:5700');
+
+    ws.onmessage = (event) => {
+      const message = JSON.parse(event.data);
+
+      if (message.roomId === roomId) {
+        setMessages((state) => [...state, message]);
+      }
+    };
+
+    ws.onerror = () => {
+      notification('error', 'Connection is lost');

```

Comment:
connection to websocket should be restored when user connection is back, please check this case (disable WiFi => wait a 30 seconds => enable WiFi => check that everything work as expected)